pkg/core/entities/notification_event.go

```diff
@@ -39,11 +39,19 @@ func (ne *NotificationEvent) DecodeBinary(r *io.BinReader) {
 // EncodeBinary implements the Serializable interface.
 func (aer *AppExecResult) EncodeBinary(w *io.BinWriter) {
 	w.WriteLE(aer.TxHash)
+	w.WriteLE(aer.Trigger)
+	w.WriteString(aer.VMState)
+	w.WriteLE(aer.GasConsumed)
+	w.WriteString(aer.Stack)
```

Comment:
You need to bump the database version after this change, because it's not backwards-compatible.

Reply:
heh